Project_Name/Readme.md

```diff
@@ -1,32 +1,15 @@
+# Basic Sample Hardhat Project
```

Comment:
I think we should keep the original README since we'll need to fill out those fields eventually anyway. If you want, you can append this section to the end